bootstrap/src/deq.ml

```diff
@@ -9,7 +9,7 @@ let c = 2
 
 let empty = (
   0, lazy Stream.Nil, lazy Stream.Nil,
-  0, lazy Stream.Nil, lazy Stream.Nil
+  1, lazy Stream.Nil, lazy Stream.Nil
```

Comment:
This looks to be unintentionally included in the pull request.  Resubmit without this change?  I'll review the rest of the code later this afternoon.